### PR TITLE
Base tests added

### PR DESCRIPTION
**Pull request by maxPakin**

### CODE REVIEW

File: src/AliceInventory/Controllers/ValuesController.cs [hunk 1]
```diff
@@ -3,43 +3,21 @@
...
-        public void Put(int id, [FromBody] string value)
-        {
-        }
-
-        // DELETE api/values/5
-        [HttpDelete("{id}")]
-        public void Delete(int id)
+        public ActionResult<string> Post([FromBody] AliceRequest request)
         {
+            var responce = new AliceResponse() {Response = new ResponseModel() {Text = request.Request.Command}};
```
Comment (ZalkinV) [inline src/AliceInventory/Controllers/ValuesController.cs]: Typo in variable name "responce". Have to be "responSe". The same mistakes exist in other places, for example, in methods' signatures

File: src/AliceInventory.IntegrationTests/PostTest.cs [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+        private StringContent CreateJsonContent(string text)
+        {
+            AliceRequest response = new AliceRequest() { Request = new RequestModel() { Command = text }};
+            string json = JsonConvert.SerializeObject(response);
+            return new StringContent(json, Encoding.Default, "application/json");
+        }
+        
+        [Theory, Order(1)]
+        [InlineData("������ 3 ������", new []{ "������ ����� 3 ��" })]
```
Comment (ZalkinV) [inline src/AliceInventory.IntegrationTests/PostTest.cs]: What type of charset you have used? I suppose, that it is not the UTF-8, therefore I have problems to see correct text in Russian inside VS Code and GitHub
image

File: src/AliceInventory/Controllers/ValuesController.cs (hunk 1 above)
Comment (maxPakin) [reply src/AliceInventory/Controllers/ValuesController.cs]: Already [fixed](https://github.com/vkamiansky/hackathonfls2019/commit/3bbd0963957d605cba5661a004e774b18f03db88)

File: src/AliceInventory.IntegrationTests/PostTest.cs (hunk 2 above)
Comment (maxPakin) [reply src/AliceInventory.IntegrationTests/PostTest.cs]: Already [fixed](https://github.com/vkamiansky/hackathonfls2019/commit/3bbd0963957d605cba5661a004e774b18f03db88)

**Pull request merged**